packages/pattern-picker/src/index.tsx

```diff
@@ -1,24 +1,43 @@
 import { Gridicon } from '@automattic/components';
+import { useStarterDesignsQuery } from '@automattic/data-stores';
+import { useLocale } from '@automattic/i18n-utils';
 import { Button } from '@wordpress/components';
 import { Icon, chevronLeft, chevronRight } from '@wordpress/icons';
 import { useI18n } from '@wordpress/react-i18n';
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useMemo } from 'react';
 import Swiper from 'swiper';
-import { PATTERN_SOURCE_SITE_SLUG } from './constants';
 import { Item } from './item';
-import useQueryPatterns from './use-query-patterns';
 import type { Pattern } from './types';
+import type { Design } from '@automattic/design-picker';
 import 'swiper/dist/css/swiper.css';
 
-type Props = { onPick: ( pattern: Pattern ) => void };
+type Props = { onPick: ( design: Design ) => void };
 
 export default function PatternPicker( { onPick }: Props ) {
 	const { __ } = useI18n();
-	const { data: patterns } = useQueryPatterns( PATTERN_SOURCE_SITE_SLUG );
+	const { data: allDesigns } = useStarterDesignsQuery( {
+		vertical_id: '',
+		intent: '',
+		_locale: useLocale(),
+		include_pattern_virtual_designs: true,
+	} );
+
+	const designs = useMemo( () => {
```

Comment:
Are the designs still ordered randomly but in a stable order for a given user like before?

Reply:
They weren't, nice catch! Fixed in https://github.com/Automattic/wp-calypso/pull/75831/commits/cc8db3367af51b42eb3a6cdb938a2711ef505113.